lib/saml20.js

```diff
@@ -123,20 +124,26 @@ exports.create = function(options, callback) {
   if (options.nameIdentifierFormat) {
     nameID.setAttribute('Format', options.nameIdentifierFormat);
   }
-  
+
   if( options.authnContextClassRef ) {
     var authnCtxClassRef = doc.getElementsByTagName('saml:AuthnContextClassRef')[0];
     authnCtxClassRef.textContent = options.authnContextClassRef;
   }
 
   var token = utils.removeWhitespace(doc.toString());
-  sig.computeSignature(token, options.xpathToNodeBeforeSignature || "//*[local-name(.)='Issuer']");
+  if (!options.location) {
```

Comment:
if `options.location` was specified without `reference` field, we should add `"//*[local-name(.)='Issuer']"` as default